src/create-user/models/index.ts

```diff
@@ -52,8 +52,8 @@ export interface UserForm {
   "What level(s) would you be comfortable mentoring at?": MentoringLevel[]; // user form
   "Where did you hear about us?": HearAboutUs[]; // user form
   "Why would you like to become a Mentor or Volunteer?": string; // user form
-  "We operate out of two locations, where would you prefer to mentor or volunteer?": Location; // user form
-  "Our homework club runs from 10:00 AM to 12:00 PM every Saturday during school term-time. How often do you think you will be able to attend? ": Frequency[]; // user form
+  "We operate out of the following locations, where would you prefer to mentor or volunteer?": Location; // user form
+  "Our homework club runs from 10: 00 AM to 12: 00 PM every Saturday during school term-time. How often do you think you will be able to attend? ": Frequency[]; // user form
```

Comment:
does the example_formdata.json need to be updated with these changes as well?

Also, where was this changed? I can't see this text in the achievers-admin app. Is there a second site where users enter this data?